aspnetcore/migration/webapi.md

```diff
@@ -28,15 +27,15 @@ In *Global.asax.cs*, a call is made to `WebApiConfig.Register`:
 
 [!code-csharp[](webapi/sample/ProductsApp/Global.asax.cs?highlight=14)]
 
-`WebApiConfig` is defined in the *App_Start* folder. It has just one static `Register` method:
+The `WebApiConfig` class is found in the *App_Start* folder and has a static `Register` method:
 
-[!code-csharp[](webapi/sample/ProductsApp/App_Start/WebApiConfig.cs?highlight=15-20)]
+[!code-csharp[](webapi/sample/ProductsApp/App_Start/WebApiConfig.cs)]
 
 This class configures [attribute routing](/aspnet/web-api/overview/web-api-routing-and-actions/attribute-routing-in-web-api-2), although it's not actually being used in the project. It also configures the routing table, which is used by ASP.NET Web API. In this case, ASP.NET 4.x Web API expects URLs to match the format `/api/{controller}/{id}`, with `{id}` being optional.
 
-The *ProductsApp* project includes one controller. The controller inherits from `ApiController` and exposes two methods:
+The *ProductsApp* project includes one controller. The controller inherits from `ApiController` and contains two actions:
```

Comment:
Do you want an API link on `ApiController`? ... I think this is the first mention of it here.

Reply:
I didn't link to it because that's an API in ASP.NET 4.x. People reading this doc will already be very familiar with `ApiController`.